cpp/src/robotick/framework/Engine.cpp

```
@@ -60,16 +60,28 @@ namespace robotick
 	extern "C" void robotick_force_register_vec3_types();
 	extern "C" void robotick_force_register_quat_types();
 
+	void align_workloads_cursor_for_type(const TypeDescriptor& type, size_t& workloads_cursor)
```

Clang (14.0.6)

[warning] parameter 'type' is unused

(misc-unused-parameters)